services/api/service.go

```diff
@@ -2228,6 +2269,80 @@ func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Reque
 	w.WriteHeader(http.StatusOK)
 }
 
+func (api *RelayAPI) saveBlockSubmissionMetrics(pf common.Profile) {
+	metrics.SubmitNewBlockCount.Add(
```

Comment:
**issue:**

this will panic, as `metrics.SubmitNewBlockCount` is not initialised.

Reply:
removed in favour of the latency metric